src/main/java/com/internet/shop/dao/jdbc/ProductDaoJdbcImpl.java

```diff
@@ -0,0 +1,122 @@
+package com.internet.shop.dao.jdbc;
+
+import com.internet.shop.dao.ProductDao;
+import com.internet.shop.exception.DataProcessingException;
+import com.internet.shop.lib.Dao;
+import com.internet.shop.model.Product;
+import com.internet.shop.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+@Dao
+public class ProductDaoJdbcImpl implements ProductDao {
+
+    @Override
+    public Product create(Product element) {
+        String insertProductQuery
+                = "INSERT INTO products (name, price, deleted) VALUES (?, ?, ?);";
```

Comment:
we won't create deleted products, it doesn't make sense 
```suggestion
                = "INSERT INTO products (name, price) VALUES (?, ?);";
```